testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1643,10 +1646,24 @@ fn test_operator_key_rotation_recovery() {
     let vm_status = result.unwrap();
     assert_eq!(VMStatusView::Executed, vm_status);
 
+    // Verify that the operator key was updated on-chain
+    let mut storage: Storage = (&backend).try_into().unwrap();
+    let operator_account_string = storage
+        .get(OPERATOR_ACCOUNT)
+        .unwrap()
+        .value
+        .string()
+        .unwrap();
```

Comment:
It's times like this, that Rust did what Kotlin did for throwable unwraps `!!` cause it's shorter :P